src/resolvers/Mutation/updateOrganization.ts

```diff
@@ -22,13 +23,22 @@ export const updateOrganization: MutationResolvers["updateOrganization"] =
     // checks if the current user is an admin of the organization
     await adminCheck(context.userId, organization);
 
+    let uploadImageFileName;
+    if (args.file) {
+      uploadImageFileName = await uploadEncodedImage(
+        args.file,
+        organization?.image
+      );
+    }
+
     return await Organization.findOneAndUpdate(
       {
         _id: organization._id,
       },
       {
         $set: {
           ...args.data,
+          image: uploadImageFileName || undefined,
```

Comment:
I've heard that prior to mongoose v6, if you use `undefined` in update fields, mongoose will set those fields to null in database instead of just ignoring them. If this is an intended behaviour ignore this comment.